src/home/HomeCookieSnackbar.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {Button, Snackbar, IconButton, useMediaQuery} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import Slide, { SlideProps } from '@mui/material/Slide';
+import theme from '../theme';
+
+
+type CookieSnackbarProps = 
+{
+    timeToWait: number;
+}
+
+type TransitionProps = Omit<SlideProps, 'direction'>;
+
+function TransitionUp(props: TransitionProps) {
+    return <Slide {...props} direction="up" />;
+}
+
+const HomeCookieSnackbar = ({timeToWait} : CookieSnackbarProps) =>
+{
+    const isDesktop = useMediaQuery(theme.breakpoints.down('lg'));
+    const [open, setOpen] = React.useState(false);
+    const [cookieMode, setCookieMode] = React.useState<boolean>();
+
+    const handleClick = (SelectedMode: boolean) => () => 
+    {
+        setOpen(false);
+        setCookieMode(SelectedMode);
+    }
+
+    const cookieAction = (
+        <React.Fragment>
+          <Button color="primary" size="small" onClick={handleClick(true)}>
+            Accept All
+          </Button>
+          <Button color="inherit" size="small" onClick={handleClick(false)}>
+            Reject All
+          </Button>
+        </React.Fragment>
+      );
+
+    React.useEffect(() => {
+        const timer = setTimeout(() => {
+            //console.log('This will open after 2 seconds!')
+            setOpen(true)
+            console.log(`Passed open after true hopefully: ${open}`);
+        }, timeToWait);
+
+        return () => clearTimeout(timer);
+      }, []);
+
+      const messageBlock = (
+        <div style={{
+            display: 'flex',
+            position: 'relative',
+            width: '100%',
+            alignItems: 'top',
+          }}>
+            <div>
+                <p>We use cookies to enhance your browsing experience and analyze our traffic. By clicking "Accept All", you consent to our use of cookies.</p>
+            </div>
+            <div>
```

Comment:
the icon placement looks a bit off. since the parent is in the relative position, we can set this to the absolute position, set a higher z-index, and set the right/top placements a little closer to the edge. basically, add a styling prop to this div and see how that looks:

`style={{ position: 'absolute', zIndex: 10, right: -10, top: -10 }}`